### DISCUSSION TITLE
Joomla4 Frontend Editing of Articles - Existing Article Content Not Populated

### DISCUSSION THREAD

**Discussion by laybergerF** [joomla/joomla-cms#36540] [General] [answered]
Hi, We are building a new site founded on Joomla 4, and would like to only use core extension like the Article features built into Joomla 4, but we've run into a wall trying to implement editing an article from the Frontend of the site. We've found an effective way to get an article that we want to edit to display within the Article Editor on the Frontend except it fails to come up with the content included in the original (it comes up blank). What do we need to do to get the content of the selected, existing article to be included for editing on the Frontend of the site? I've included screen captures showing how we get the Event Editor to display on the Frontend and the resulting, blank, unpopulated editor.

![ArticleEditig-NoPopulatingOfContent-2022-01-02-10_17_27](https://user-images.githubusercontent.com/5634531/147887726-29900642-ce7c-4e67-be5c-a73e0882a2d7.jpg)
.

**Comment by brianteeman** [comment]
Without knowing what your "effective way" is it is impossible to say

**Reply by laybergerF** [reply]
I'm sorry that I wasn't clear. The 'Effective Way' that we're using to get an article to display in the article editor on the Frontend is what I was attempting to show with the image. when the user is logged into the site, we display a menu item which opens up a list of article categories. Selecting one of the categories brings up all of the articles in the chosen category with an option to 'Edit' next to each article.. Selecting the 'Edit' option brings up the article editor. The issue is that when the editor is display, it doesn't contain any of the content from the selected article, it's blank. What we're looking for is a way to have it populated with the content from the article so that a user can then edit and save the modified article.

**Reply by brianteeman** [reply]
This is a core feature. Works for me

![tables3](https://user-images.githubusercontent.com/1296369/147956105-04edb7aa-aba4-4c9e-9e96-6d29fa9901ad.gif)

**Reply by laybergerF** [reply]
Thank you.... It looks like we need to pursue this issue with our template developer. We're using the T4 Framework with their t4_bs5_blank template.

**Comment by laybergerF** [comment]
To see what effect the T4 Framework/t4_bs5_blank template might be having on this issue, I disabled all of the T4 related components and the t4_bs5_blank template. I then set the Cassiopeia template as the default template for the site. Doing so changed the appearance of the site, but not the issue of the article editor opening open void of any content from the article selected for editing. I've attached images showing the issue using the Cassiopeia template. Logged into the site as a super user, any of the articles on the page can be selected for editing by clicking on the 'Edit' button, which then opens the article editor, but without any content from the selected article. 

![ArticleEditig-NoPopulatingOfContent-T4disabled-CassiopiaTemplatei](https://user-images.githubusercontent.com/5634531/147960755-942f5dcb-8599-45c8-9d14-52bd1b37b065.jpg)

**Comment by brianteeman** [comment]
dont know what to say. it even works for me with the same template as you.

Check that there are no javascri0pt errors in your browser console

**Reply by laybergerF** [reply]
Here's what's being displayed in the Console: 
Error with Permissions-Policy header: Unrecognized feature: 'interest-cohort'.
jquery-migrate.min.js?3.3.2:2 JQMIGRATE: Migrate is installed, version 3.3.2

2joomla-field-media.min.js?b9214e72c06daafd562e5c4116292b5763a437fe:1 
        
       Uncaught TypeError: window.bootstrap.Modal.getInstance is not a function
    at HTMLElement.connectedCallback (joomla-field-media.min.js?b9214e72c06daafd562e5c4116292b5763a437fe:1)
    at joomla-field-media.min.js?b9214e72c06daafd562e5c4116292b5763a437fe:1
connectedCallback @ joomla-field-media.min.js?b9214e72c06daafd562e5c4116292b5763a437fe:1
(anonymous) @ joomla-field-media.min.js?b9214e72c06daafd562e5c4116292b5763a437fe:1

DevTools failed to load source map: Could not load content for https://coloradosquaredance.com/Test/media/vendor/jquery-migrate/js/jquery-migrate.min.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

**Reply by brianteeman** [reply]
well those errors are the reason for your problem. Specifically the joomla-field error. 

Please use the forum for support https://forum.joomla.org

**Comment by laybergerF** [comment]
Okay, I've set a menu item for a single article and set it as the default page. With T4 uninstalled and the Cassiopeia template set as the default for all pages, selecting 'Edit' still brings up the Frontend article editor void of content.

**Comment by laybergerF** [comment, marked as answer]
Resolved: With a fresh install of Joomla 4 on my localhost system I found that I could add / edit an article from the Frontend. so I tried copying all of the files from the fresh install of Joomla 4 over top of the existing files on our test site. It resolve the issue. We can now add and edit files via the Frontend editor. Thank you for your help.
